### DISCUSSION TITLE
so-analyst-install command not found

### DISCUSSION THREAD

**Discussion by Todd998** [Security-Onion-Solutions/securityonion#5505] [Unsupported Versions] [answered]
Hi all, I've installed SO from the ISO with CentOS 7. I have not installed Security Onion as all I want is the Analyst VM however the script won't run. Another thread suggested mounting the ISO and running it from there. I tried that to no avail. 

Any suggestions?

Thank you in advance.

**Comment by dougburks** [comment, marked as answer]
If you've installed our ISO image but haven't run through Setup, then our scripts have not yet been copied from your home directory into /usr/sbin/. You should be able to run `sudo SecurityOnion/so-analyst-install`. Please note that this requires Internet access, so you will either need to configure networking manually or run our Setup and choose the `Configure Network` option.

**Reply by Todd998** [reply]
Perfect. That did it.
